charts/zookeeper/values.yaml

```diff
@@ -29,7 +29,10 @@ pod:
   # tolerations: []
   # env: []
   # annotations: {}
-  # securityContext: {}
+  # securityContext:
+    #runAsUser: 1000
+    #runAsGroup: 1000
+    #runAsNonRoot: false
```

Comment:
I don't think `runAsNonRoot` is used anywhere.  We should probably remove the comment here to avoid confusion.

Reply:
Changed